packages/voice/src/models/v1/call-object/call-object.ts

```diff
@@ -0,0 +1,31 @@
+import { ReasonEnum, ResultEnum } from '../enums';
+import { VoicePrice } from '../voice-price';
+import { Participant } from '../participant';
+
+export interface CallObject {
```

Comment:
It took me some time to identify what mean "CallObject" and it usage.
Finally have to ensure I was right by looking at the OAS file details to identify it is the `getCallResponseObj` schema.
So finally, may be it could mean the name should convey more information for auto-documentation  purpose and readability for maintainers

Reply:
Naming is a complicated task :)
The main object here is a 'Call' which is also the name of the service 'Calls' and also the verb 'call'. This is why I've added the suffix 'Object' to be clear about which kind of call it was and to avoid a naming conflict.
Then, the operationId is `Calling_GetCallResult` but here "Result" doesn't mean "Response" but rather "Status" which confused me too. The response object defined in the specification is `getCallResponseObj` which is "the object that represents the response of a getCall (status) query". So I thought to name it "CallResponse" but then it would be confusing with the actions of the "Call" service.
Long story short: what do you think about `CallInformation`?

Reply:
Yes; and it is not so far from documentation related to operation: `Get information about a call`

Reply:
Fixed. In the end it's `GetCallInformation` as it is the result of a `GET` call, such as "get callbacks" for instance